src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/SyntaxEditorBasedCodeFixProvider.cs

```diff
@@ -13,10 +13,16 @@ namespace Microsoft.CodeAnalysis.CodeFixes
 {
     internal abstract partial class SyntaxEditorBasedCodeFixProvider : CodeFixProvider
     {
+        private static readonly ImmutableArray<FixAllScope> s_defaultSupportedFixAllScopes =
+            ImmutableArray.Create(FixAllScope.Document, FixAllScope.Project, FixAllScope.Solution,
+                FixAllScope.ContainingMember, FixAllScope.ContainingType);
+
         private readonly bool _supportsFixAll;
 
         protected SyntaxEditorBasedCodeFixProvider(bool supportsFixAll = true)
-            => _supportsFixAll = supportsFixAll;
+        {
+            _supportsFixAll = supportsFixAll;
+        }
```

Comment:
unnecessary change.  but fine if you prefer the block form.

Reply:
I had edited the ctor to add another statement and then reverted, but forgot to switch the code back to expression body form.